.github/CONTRIBUTING.md

```diff
@@ -76,15 +86,11 @@ Before you issue a pull request that modifies code:
 
 Before you start making fundamental changes:
 
-- Propose it by creating a [github issue](https://github.com/ElektraInitiative/libelektra/issues/new)
-  for discussions.
 - You should read the [design document](/doc/DESIGN.md).
 - Create a [decision](/doc/decisions/README.md) describing you want to do
   to keep the discussion more efficient and architecture documented.
 
 # Labels
 
-- If you do not want your PR to be merged, please label
-  the PR with the label "work in progress".
-- If you included changes after a review and you think you
-  are ready, please tag your PR with the label "ready to merge".
+- If you do not want your PR to be merged, please label the PR with the label "work in progress".
+- If you included changes after a review and you think you are ready, please tag your PR with the label "ready to merge".
```

Comment:
Hmm maybe also re-request a review? I do not get notified if a label changes.

Reply:
Yes, label changes are silent. I'd say definitely re-request a review from:
- all previous reviewers, if you made substantial changes
- all reviewers who rejected, if you just applied the changes requested in the reviews or fixed build errors.